internal/glfwkeyconversion/map-keycode-to-macos.go

```diff
@@ -0,0 +1,559 @@
+package glfwkeyconversion
+
+import "github.com/go-gl/glfw/v3.3/glfw"
```

Comment:
we may be better served by making this platform specific.
```
internal/keyboard/keyboard_darwin.go // has all the mappings defined in these two files.
internal/keyboard/keyboard_unix.go // with a +build unix !darwin
internal/keyboard/keyboard_windows.go
```

each file with a single method:

func Normalize(keycode glfw.Key, scancode int) Event { /* ... */ }

```golang
// internal/keyboard/keyboard.go
// replaces keyEventMessage
type Event struct {
  // ...
}
```